notebooks/hyperkitty_to_csv.ipynb

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
<p>Is anything below this cell needed? It looks like its all duplicate of the cells above.</p><p> </p><p>Also does this notebook include things from <code>retrieve_mbox</code>and  <code>parsing_mbox</code>? If so, are the other notebooks necessary any longer?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/fedora-mailing-list-analysis/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>